website/docs/aggregation operations/aggregation-stages.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 2
+slug: /aggregation-stages/
+---
+
+# Aggregation stages
+
+Aggregation stages are a series of one or more processes in a pipeline that acts upon the returned result of the previous stage, starting with the input documents.
+
+| Supported aggregation stages | Description                                                                                           |
```

Comment:
Both stages don't show up under [supported commands](https://docs.ferretdb.io/reference/supported-commands/#aggregation-pipeline-stages). I'll send another PR to fix that and update this one subsequently

Reply:
Hmm, it seems that with `$project` we forgot to mark it implemented (the issue is closed).
In case of `$collStats` it's partly implemented - the stage is supported, it just doesn't show all the possible data. 
Let's have a separate PR with these stages then, yes.